docs/quiz-toolkit/overview.md

```diff
@@ -0,0 +1,21 @@
+---
+sidebar_label: "Overview"
+sidebar_position: 1
+---
+# Overview
+
+
+**MecSimCalc** now supports online quizzes, allowing professors to efficiently administer quizzes and automatically collect data. This new functionality enriches MecSimCalc's capabilities, enabling dynamic input generation where quiz values can be **randomized**, and **correctness is determined through logical code evaluation**.
+
+Currently, MecSimCalc offers two convenient methods for data collection:
```

Comment:
Remove most of this and incorporate the "Google Sheets" and "Gmail" documents into one whole tutorial